content/en/blog/2023-03-31-helm-completes-fuzzing-security-audit.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Helm Completes Fuzzing Security Audit"
+slug: "helm-completes-fuzzing-security-audit"
+authorname: "Adam Korczynski, David Korczynski & Martin Hickey"
+author: "@AdamKorcz, @DavidKorczynski & @hickeyma"
+authorlink: "https://helm.sh"
+date: "2023-03-31"
+---
+
+In the past year, the team at [Ada Logics](https://adalogics.com) has worked on integrating continuous fuzzing into the [Helm core project](https://github.com/helm/helm). This was an effort focused on improving the security posture of Helm and ensuring a continued good experience for Helm users. The fuzzing integration involved enrolling Helm in the [OSS-Fuzz project](https://github.com/cncf/cncf-fuzzing) and writing a set of fuzzers that further enriches the test coverage of Helm. In total, 38 fuzzers were written, and nine bugs were found (eight fixed), demonstrating the work’s value for Helm both short term and long term. All fuzzers were implemented by way of [Go-fuzz](https://github.com/dvyukov/go-fuzz) and are run daily by OSS-Fuzz against the latest Helm commit to make sure Helm is continuously fuzz tested. The full report of the engagement can be found [here](https://github.com/helm/community/tree/main/security-audit/FUZZING_AUDIT_2022.PDF). 
```

Comment:
Do we want to clarify that the one bug we found but didn't yet fix does have a fix in progress? Or at least say "nine bugs were found (with eight fixed so far)"?

Reply:
Reads better with sentence update. Updated.